Yoast/wordpress-seo
# Use recommended test reporter method for CodeClimate

## Summary

Implements the Travis CI coverage method as per following example: https://docs.codeclimate.com/docs/travis-ci-test-coverage

## Test instructions

This PR can be tested by following these steps:

* Make sure the build passes and coverage is updated in Code Climate

## Quality assurance
